define the linear ordering when SuiteSparse is not used
(otherwise chokes on a unit test)
bicatali@gentoo.org

--- ceres-solver-1.4.0.orig/internal/ceres/solver_impl_test.cc	2012-11-19 13:34:36.194795640 -0800
+++ ceres-solver-1.4.0/internal/ceres/solver_impl_test.cc	2012-11-19 13:59:42.395429059 -0800
@@ -498,6 +498,8 @@
 #if defined(CERES_NO_SUITESPARSE) && defined(CERES_NO_CXSPARSE)
 TEST(SolverImpl, CreateLinearSolverNoSuiteSparse) {
   Solver::Options options;
+  // CreateLinearSolver assumes a non-empty ordering.
+  options.linear_solver_ordering = new ParameterBlockOrdering;
   options.linear_solver_type = SPARSE_NORMAL_CHOLESKY;
   string error;
   EXPECT_FALSE(SolverImpl::CreateLinearSolver(&options, &error));
